tests/python/test_api.py

```diff
@@ -209,7 +209,7 @@ def _get_expected_matrix_apis():
     "simt",
     "sin",
     "solve",
-    "sparse_matrix_builder",
+    "sparse" "sparse_matrix_builder",
```

Comment:
You're missing a comma here...
```suggestion
    "sparse",
    "sparse_matrix_builder",
```